[diff_drive_controller] Fix missing cmath header in speed_limiter.cpp

olivier-stasse opened the pull request:
Fix  #147

bmagyar commented:
Could you please also remove the include you mention on math.h and check if it's still OK? The CI only runs Ubuntu builds

olivier-stasse commented:
> Could you please also remove the include you mention on math.h and check if it's still OK? The CI only runs Ubuntu builds

Done. To make it simpler I changed `math.h` in `cmath` in `speed_limiter.hpp` and removed the inclusion I added in `speed_limiter.cpp`.
 It is compiling on macOS Catalina.

bmagyar commented:
perfect, thank you!
